### ISSUE TITLE
Not working in Phpstorm 2016.2

### ISSUE THREAD

**Issue by carlosflorencio** [bashor/IdeaMainMenuToggler#2]
Hi,

Can you fix it? 

Thanks

**Comment by bashor** [comment]
Could you please clarify what kind of problem do you have?
Can't install?
It doesn't work silently?
Could you send log files after trying to use the action? ([Locating IDE log files](https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files))

**Comment by carlosflorencio** [comment]
It doesn't work silently.

First the error was in the log: 'Plugin does not include any module dependency tags in its plugin.xml therefore is assumed legacy and can be loaded only in IntelliJ IDEA' so the plugin won't load.

I added `<depends>com.intellij.modules.platform</depends>` to the plugin.xml and the plugin loaded fine.

But if i toggle the action to hide the main menu and restart the phpstorm the menu is always there.

Thanks

**Comment by bashor** [comment]
Nice, thank you!

> But if i toggle the action to hide the main menu and restart the phpstorm the menu is always there.

It doesn't store and restore state, You should hide the menu bar each time after load, it was good enough for me.
Feel free to contribute ;)

**Comment by carlosflorencio** [comment]
I can't make it work with the plugin enabled.

The main menu is always there.

![phpstorm_2016-07-14_23-11-02](https://cloud.githubusercontent.com/assets/1500881/16857551/612526d6-4a18-11e6-8899-d74b6571b65d.png)

**Comment by bashor** [comment]
Could You please try [this version](https://www.dropbox.com/s/2spl33nl5i8j01n/IdeaMainMenuToggler.zip?dl=0)
Right now I'm far from good internet and can't download PHPStorm to check.

**Comment by carlosflorencio** [comment]
Yep that version works, but the setting is not persisted.

Can you update this repo with those changes?

The kotlin libraries are necessary? 

From http://www.jetbrains.org/intellij/sdk/docs/tutorials/kotlin.html:

> Plugins targeting the IntelliJ Platform versions 143 and above are easy to migrate: just start writing Kotlin. The necessary Kotlin plugins and libraries are already bundled with the IDE, requiring no further configuration. For version 142 or below, you will need to install and configure dependencies to the Kotlin runtime (in addition to installing the Kotlin plugin itself, for code assistance and tooling support). For detailed instructions, please refer to the Kotlin documentation.

Thanks!

**Comment by bashor** [comment]
> Yep that version works, 

Thanks for the testing!

> but the setting is not persisted.

It's expected. 
Unfortunately, I no longer use this plugin and don't plan to spend much time on its development.

> Can you update this repo with those changes?

I've pushed changes to the repo and add the new version to the plugin repository.

> The kotlin libraries are necessary?

Yep, it should be available at runtime., but a user should not do anything for that.

> Plugins targeting the IntelliJ Platform versions 143 and above are easy to migrate: just start writing Kotlin....

It's mostly about development plugins in Kotlin.
Anyway thanks for the comment, I've increased minimal platform version (to simplify build process and to stay plugin small as possible).
